### DISCUSSION TITLE
Contribution Pledge to Oqtane Documentation 🚀

### DISCUSSION THREAD

**Discussion by thabaum** [oqtane/oqtane.docs#44] [General]
# Contribution Pledge to Oqtane Documentation 🚀

Hello Oqtane Documentation Contributors! 🚀 

I'm excited to share my commitment to contribute and assist everyone in learning how to use the Oqtane Framework, both as an end user and developer.

## About Me
Some of you may know me from my involvement with the project since beta 0.8. I've been actively providing opinions on our documentation and its improvement. The documentation has been scattered across various platforms, including here, blogs, and an upcoming wiki.

Now that Oqtane has reached a very stable level, I believe it's crucial to create a "hub" at Learn.Oqtane.org that connects us to valuable solutions provided by community members. This can include videos, shows spotlighting Oqtane, tutorials, and comprehensive documentation.

## Contribution Approach
I will be prefixing my GitHub issues with "Doc:" to highlight areas that need attention in the documentation. If you're interested in working on a specific issue, please feel free to announce your intent. If you find it challenging, kindly let us know if you've decided not to proceed, so others can consider taking it up.

As a personal pledge, I'll be diligently working over the next month to enhance the documentation. I'll focus on providing detailed instructions, accompanied by images, for all installation types based on the current ones available.

## Community Collaboration
When there are behavior changes or enhancements made to the framework, it's our collective responsibility to log those changes in GitHub issues. This ensures that someone can attempt to address the issue by submitting a related pull request.

Several sections in the documentation are blank, making them open for pull requests. Feel free to create an issue, stating your intention to work on it. Submit the pull request when finished. While some prefer not to create an issue until they have a pull request ready, it's beneficial to know that someone is actively working on a particular task.

Let's collaborate and make the Oqtane documentation a comprehensive and reliable resource for the entire community!

Thank you, Cheers! 🍻

**Comment by thabaum** [comment]
Last month I pretty much spent a lot of time clearing up things for other contributors while figuring things out for myself.   The documentation is tailored for a different approach than what I generally use and I wanted to be sure these instructions where enhanced as well as needed.

Things I accomplished as contributions towards the Oqtane.Docs repository over the past month are the following:
- Spelling and Typo Corrections - PRs #56 and #52 
- Added media creativity and contribution section - PR #53
- Visual Studio 2022 docs guide enhancement handling build errors from the zip download approach - PR #51

These are nothing major, but can help others wishing to contribute as I do with some added clarity and tips.

Revisiting these issues sorta jammed me up from my main objectives.  I do have a project to share what I had in notes I generated over the past month that covers Linux Ubuntu, Databases, Apache, IIS, Windows topics for installing Oqtane.  I even created a developer section but naming things is not in concrete as this is not even in draft form yet.  

When it is ready for complete review I will put it up as a draft hoping for some test runs making sure anything we need can be added.  My goal is to make it able to merge so everyone can enhance them further as felt needed.

Here is the link to my repository until removed:  https://github.com/thabaum/oqtane.docs/tree/installation-demo

Feel free to review and ask any questions here.   I will close this thread once this is completed as in merged into docs.  I am at the point of fine tuning this installation enhancement to be proposed by adding images as I walk through them from fresh, scratch installations of each operating system used.  I will use windows 2022 server, Windows 11 and Linux Ubuntu for now.

Cheers!

**Comment by iJungleboy** [comment]
awesome.
feel free to push, even if still 80% done ;)

**Reply by thabaum** [reply]
@iJungleboy just pushed what I have currently to review, make suggestions as needed.  We can discuss more at upcoming meeting.  PR #57

**Reply by thabaum** [reply]
well... i said by year end we would have Oqtane 5.x.x documented... just in time for Oqtane 6.0.0 :)

:rocket:

I am 100% done... moving forward you can close most all my issues and a few of the others.  Next meeting we will close the rest.

#91 #82 #88  #84 #80 ... should finally put my money where my mouth has been ;)
